### DISCUSSION TITLE
Increase "Original URL" columns character limit

### DISCUSSION THREAD

**Discussion by TritonB7** [YOURLS/YOURLS#3107] [Q&A] [answered]
How do I increase the character limit placed on the information show in the "Original URL" column? Anything past a certain length get's cut replaced with the following [...].

![CharacterLimit1](https://user-images.githubusercontent.com/31591982/140067931-c92f07e2-5e8f-4b7b-b6e0-a6ffb7234f3c.jpg)

**Comment by dgw** [comment, marked as answer]
The long URL for display (`long_url_html`) goes through `yourls_trim_long_string()`:

https://github.com/YOURLS/YOURLS/blob/83531acb0f8ab642a728c741f5203b4241dcd81c/includes/functions-html.php#L594-L601

and `yourls_trim_long_string()` has a `trim_long_string` filter you could hook into [with a plugin](https://github.com/YOURLS/YOURLS/wiki/How-to-make-Plugins) to override the default behavior:

https://github.com/YOURLS/YOURLS/blob/83531acb0f8ab642a728c741f5203b4241dcd81c/includes/functions-formatting.php#L194-L200

A `$string` parameter starting with something that looks like a protocol with the default `$length` of 60 and default `$append` of `'[...]'` is about as close as you'll get to a context check, I think. There's no specific filter, configuration value, or other means of just directly telling YOURLS to show more of the full link.
